test/hooks_test.dart

```diff
@@ -407,6 +407,85 @@ main() {
         });
       });
     });
+
+    group('useRef -', () {
+      var mountNode = new DivElement();
+      ReactDartFunctionComponentFactoryProxy UseRefTest;
+      ButtonElement reRenderButton;
+      var noInitRef;
+      var initRef;
+      var domElementRef;
+      StateHook<int> renderIndex;
+      var refFromUseRef;
+      var refFromCreateRef;
+
+      setUpAll(() {
+        UseRefTest = react.registerFunctionComponent((Map props) {
+          noInitRef = useRef();
+          initRef = useRef(mountNode);
+          domElementRef = useRef();
+
+          renderIndex = useState(1);
+          refFromUseRef = useRef();
+          refFromCreateRef = react.createRef();
+
+          if (refFromUseRef.current == null) {
+            refFromUseRef.current = renderIndex.value;
+          }
+
+          if (refFromCreateRef.current == null) {
+            refFromCreateRef.current = renderIndex.value;
+          }
+
+          return react.Fragment({}, [
+            react.input({'ref': domElementRef}),
+            react.p({}, [renderIndex.value]),
+            react.p({}, [refFromUseRef.current]),
+            react.p({}, [refFromCreateRef.current]),
+            react.button({
+              'ref': (ref) => reRenderButton = ref,
+              'onClick': (_) => renderIndex.setWithUpdater((prev) => prev + 1)
+            }, [
+              're-render'
+            ]),
+          ]);
+        });
+
+        react_dom.render(UseRefTest({}), mountNode);
+      });
+
+      group('correctly initializes a Ref object', () {
+        test('with current property set to null if no initial value given', () {
+          expect(noInitRef is Ref, isTrue);
```

Comment:
There's a matcher for this that will give you better error messages when this fails, instead of just saying `Expected: true. Actual: false`: 
```suggestion
          expect(noInitRef, isA<Ref>());
```

Reply:
Nice! Thanks!